linmod/utils.py

```diff
@@ -13,8 +13,26 @@ def expand_grid(**columns):
     return df.sort(columns.keys())
 
 
+def pl_crps(samples_column: str, truth_column: str):
+    """
+    Monte Carlo approximation to the CRPS.
+    """
+
+    samples = pl.col(samples_column)
+    truth = pl.col(truth_column)
+    n = samples.len()
+
+    return (samples - truth).abs().mean() - 0.5 * (
+        samples.head(n - 1) - samples.tail(n - 1)
+    ).abs().mean()
+
+
 def pl_softmax(pl_expr, over=None):
```

Comment:
I think you could just write this as the `if over is None` bit, because you could always do
```python
my_expr.pipe(pl_softmax).over(pl.col('group'))
```
right?

Reply:
Absolutely, and I realized this after the fact and have been wanting to change it, but decided to leave it to a separate PR to not clutter this one. But now that you point it out, I am very happy change it :)